app/models/game.rb

```diff
@@ -30,4 +32,10 @@ class Game < ActiveRecord::Base
   def complexity_name
     COMPLEXITIES[complexity]
   end
+
+  def update_average
+    s = reviews.sum(:game_rating)
+    c = reviews.count
+    update_attribute(:average, c == 0 ? 0.0 : s / c.to_f)
```

Comment:
Heroku would crash then. And I don't see how this would break. It only counts when you add a new review so I don't think it would contribute significant lag.